BLL/DrinkLoader.cs

```diff
@@ -6,21 +6,18 @@
 
 namespace BLL
 {
-    public class DrinkLoader
+    public static class DrinkLoader
     {
-        private List<Drink> _drinks;
-
-        public DrinkLoader(List<Drink> drinks)
-        {
-            _drinks = drinks;
-        }
-
-        public void AddDrinksFromFile()
+        public static List<Drink> AddDrinksFromFile(List<Drink> drinks = null)
         {
-            var newDrinks = new List<Drink>();
-            var path = Environment.CurrentDirectory + "/" + "drinks_source.json";
+            List<Drink> newDrinks;
+            string path;
 
-            if (File.Exists(path) == false)
+            if (drinks == null)
+            {
+                path = Environment.CurrentDirectory + "/" + "drinks_source.json";
+            }
+            else
```

Comment:
I still don't get it, why this class has to be static and why it gets drinks as an argument with default null value instead of just returning drinks from a file. It's not called `DrinksListManager` or something like that, It's `DrinkLoader`. It supposes to do only that, load drinks from a file. I highly recommend, instead of making static class, use it just as a loader (as the name suggests) and save results to (for now) some static list declared in the main program and then make any operations on that list. This way you will have a pure solution with behavior following the design. @wojciechkapko I believe it's your part so please take care of it.

Reply:
The intent was to use it both for initial file load (if there is no argument passed) and then to extend already existing drinks list i will separate it into two methods.